app/assets/javascripts/controllers/FileCtrl.coffee

```diff
@@ -1,14 +1,24 @@
 @bruseApp.controller 'FileCtrl', ['FileHandler', '$scope', '$http', (FileHandler, $scope, $http) ->
   $scope.files = []
+  $scope.bruse_files = []
   $scope.loading = true
 
   # get identity information from add view
   $scope.identity = IDENTITY_PARAMS
 
-  # load root files on page load
-  FileHandler.get($scope.identity.id, '/').then((data) ->
-    $scope.files = data
-    $scope.loading = false
+  # load existing BruseFiles on page load
+  FileHandler.collect($scope.identity.id).then((data) ->
```

Comment:
The `->` is the CoffesScript way to declare a function.

This code in CoffeScript:

```coffeescript
FileHandler.get($scope.identity.id, '/').then((data) ->
    $scope.files = data
    $scope.loading = false
   
```

Translates to this JavaScript:

```javascript
FileHandler.get($scope.identity.id, '/').then(function(data) {
  $scope.files = data;
  return $scope.loading = false;
});
```